data/config-schema/src/main/resources/net/corda/schema/configuration/sandbox/1.0/corda.sandbox.json

```diff
@@ -4,6 +4,7 @@
   "title": "Corda Sandbox Configuration Schema",
   "description": "Configuration schema for the sandbox section.",
   "type": "object",
+  "additionalProperties": false,
```

Comment:
you have to add this at every "level" in this file. Anywhere that has a "properties" block